light/client.go

```diff
@@ -121,6 +123,14 @@ func MaxBlockLag(d time.Duration) Option {
 	}
 }
 
+// Provider timeout is the maximum time that the light client will wait for a
+// provider to respond with a light block.
+func ProviderTimeout(d time.Duration) Option {
+	return func(c *Client) {
+		c.providerTimeout = d
+	}
+}
+
```

Comment:
Entirely cosmetic suggestion:
```suggestion
func ProviderTimeout(d time.Duration) Option {
	return func(c *Client) { c.providerTimeout = d }
}
```